processing/src/main/java/io/druid/query/lookup/LookupBean.java

```diff
@@ -19,32 +19,76 @@
 
 package io.druid.query.lookup;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Preconditions;
 
 //TODO merge this code to the same definition when pr/1576 is merged
```

Comment:
also looked at #1576 and this TODO , not sure what was the todo for @b-slim i guess you wrote this TODO, what was your intention?